bugbug/models/component.py

```diff
@@ -221,10 +221,14 @@ def get_meaningful_product_components(self, full_comp_tuples, threshold_ratio=10
         max_count = product_component_counts[0][1]
         threshold = max_count / threshold_ratio
 
+        active_product_components = bugzilla.get_active_product_components(
+            self.PRODUCTS + list(self.PRODUCT_COMPONENTS)
```

Comment:
`self.PRODUCTS` is a set, we cannot use `+` on it
```suggestion
            self.PRODUCTS | self.PRODUCT_COMPONENTS.keys()
```

Reply:
Done in 53934f579c8e463189a3b150b02b8ebe5173cfe5.